app/templates/_package.json

```diff
@@ -40,4 +40,4 @@
 			"text"
 		]
 	}<% } %>
-}
+}
```

Comment:
^

Reply:
What is this `^` mean ?

Reply:
It's an arrow pointing out:

> Don't do unrelated changes.

Reply:
Always look over the full diff after every change to catch mistakes like this.

Reply:
@sindresorhus thanks for the advice 👍